docs/core/windows-prerequisites.md

```diff
@@ -13,45 +13,117 @@ ms.assetid: c33b1241-ab66-4583-9eba-52cf51146f5a
 
 # Prerequisites for .NET Core on Windows
 
-This article shows you what dependencies you need to deploy and run .NET Core applications on Windows machines and develop using Visual Studio.
+This article shows you the needed dependencies to develop .NET Core applications on Windows machines. The supported OS versions and dependencies that follow apply to the three ways of developing .NET Core apps on Windows: via the [command line with your favorite editor](tutorials/using-with-xplat-cli.md), [Visual Studio Code](https://code.visualstudio.com/), and [Visual Studio Community 2017](https://www.visualstudio.com/downloads/).
 
-## Supported Windows versions
+[.NET Core 1.x](#tab/netcore1x)
 
-.NET Core is supported on the following versions of Windows:
+**.NET Core 1.x Supported Windows Versions**
+
+.NET Core 1.x is supported on the following versions of Windows:
 
 * Windows 7 SP1
 * Windows 8.1
 * Windows 10
 * Windows Server 2008 R2 SP1 (Full Server or Server Core)
 * Windows Server 2012 SP1 (Full Server or Server Core)
 * Windows Server 2012 R2 (Full Server or Server Core)
-* Windows Server 2016 (Full Server, Server Core or Nano Server)
+* Windows Server 2016 (Full Server, Server Core, or Nano Server)
 
 See the [.NET Core Release Notes](https://github.com/dotnet/core/blob/master/release-notes/1.1/1.1.md) for the full set of supported operating systems.
 
-## .NET Core dependencies
+**.NET Core 1.x Dependencies**
+
+.NET Core 1.x requires the Visual C++ Redistributable when running on Windows versions earlier than Windows 10 and Windows Server 2016. This dependency is automatically installed for you if you use the .NET Core 1.x installer. 
```

Comment:
This feedback is for an item that was not originally revised as part of this update. Replacing "for you if you use" with "by" is a great suggestion and I'm updating accordingly. Thanks!